### PR TITLE
Download page / mailto version

### PR DESCRIPTION
**Pull request by qooqu**
Proof of concept

### CODE REVIEW

**Comment by aahvocado** [comment]
Hmm, code-wise I don't have a problem with this. I would like it to open in a window or new tab though so the user doesn't lose the ata.

**Comment by qooqu** [comment]
There's an FF bug that ignores `target="_blank"` in mailto links: https://bugzilla.mozilla.org/show_bug.cgi?id=646552

There's a bunch of workarounds. Some get away from mailto, but I don't think that's a great idea because you lose all the default browser and system behavior. If you put the mailto in an iframe, the security rules overcome the FF bug (and it still works in Chrome).

Not sure how to pass info into the iframe, but it looks do-able: https://stackoverflow.com/questions/536538/pass-value-to-iframe-from-a-window

**Comment by qooqu** [comment]
Sent screenshots (of b6ddd4c) to Brandon who took them to the design team. They are ok with purple buttons in a column and asked us to:
- differentiate the txt button from the pdf button via the icon (they sent an example of a download icon that says pdf)
- left align the text in the buttons

**Comment by aahvocado** [comment]
I am happy with the b6ddd4ce73c9e708aa411c5915524a38e17c0276 changes (just need to clean up the commented out code). Are we waiting on approval from design team @qooqu ?

**Comment by qooqu** [comment]
discussed at 2021-10-07 dev meeting:

1. design asked for pdf and txt icons. they aren't available in mui. we'll let them know we'd need to add a library and would prefer not to
2. design asked for alignment tweaks on the buttons. i'll work on it
3. i'll clean up the code

File: react/client/src/contexts/RoutingProps.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ export function getNextFormStep(currentStep: string) {
     case STEP_ENUMS.FINALIZE:
       return STEP_ENUMS.FINALIZE_PREVIEW;
     case STEP_ENUMS.FINALIZE_PREVIEW:
-      return STEP_ENUMS.NONE;
+      return STEP_ENUMS.DOWNLOAD;
```
[the remark this answers is not in the document]
Comment (qooqu) [reply react/client/src/contexts/RoutingProps.ts:144]: Not sure I understood exactly what you meant, but I took a crack at it.

File: react/client/src/styles/utilityStyles.ts [hunk 2]
```diff
@@ -87,6 +87,23 @@ const useUtilityStyles = makeStyles<Theme>((theme) =>
...
+    downloadButtonsContainer: {
+      marginTop: '1rem',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      // justifyContent: 'flex-start',
+      // [theme.breakpoints.up(theme.breakpoints.values.md)]: {
+      //   flexDirection: 'column',
+      //   justifyContent: 'space-evenly',
+      // },
```
[the remark this answers is not in the document]
Comment (qooqu) [reply react/client/src/styles/utilityStyles.ts]: done

File: react/client/src/components/Button.tsx [hunk 3]
```diff
@@ -119,6 +124,9 @@ const ButtonComponent = ({
       className={`${classes.root} ${className}`}
       onClick={onClick}
     >
+      {icon === 'EmailIcon' ? <EmailIcon /> : null}
+      {icon === 'GetAppIcon' ? <GetAppIcon /> : null}
+      {icon === 'FileCopyIcon' ? <FileCopyIcon /> : null}
```
[the remark this answers is not in the document]
Comment (qooqu) [reply react/client/src/components/Button.tsx]: done

**Review by aahvocado — APPROVED** [review]
@qooqu That's the stuff, thanks!!! 🎉

**Pull request merged**